x/bonds/module.go

```diff
@@ -154,3 +154,13 @@ func (am AppModule) BeginBlock(_ sdk.Context, _ abci.RequestBeginBlock) {}
 func (am AppModule) EndBlock(ctx sdk.Context, _ abci.RequestEndBlock) []abci.ValidatorUpdate {
 	return EndBlocker(ctx, am.keeper)
 }
+
+// Cosmos-SDK v0.44.5 required this function. Not yet sure about the value.
+// TODO: Defaulting this to 0 for now, but need to find out if we have a different version
+// ConsensusVersion is a sequence number for state-breaking change of the
+// module. It should be incremented on each consensus-breaking change
+// introduced by the module. To avoid wrong/empty versions, the initial version
+// should be set to 1.
+func (am AppModule) ConsensusVersion() uint64 {
+	return 0
+}
```

Comment:
@ig-shaun @demondayza @steveixo 
A ConsensusVersion function is needed going forward. I based on what is said in their comments this should be 1, I have made it 0 as I did not know what the side effect of this would be. Is there a different version I should be using / supplying or should I use 1 as our initial version?

Reply:
Should be the same across the board for the base version, stick to 1.